Assets/Text/cqui_text_settings.xml

```diff
@@ -178,10 +178,10 @@
       <Text>Toggles the popup that appears whenever a new tech or civic is achieved</Text>
     </Row>
     <Row Tag="LOC_CQUI_POPUPS_TECHAUDIO" Language="en_US">
-      <Text>Tech/Civic popup audio</Text>
+      <Text>Auto-play Tech/Civic popup audio</Text>
```

Comment:
To avoid confusion for non-English people, pls. avoid "popup audio".
e.g. "Play the audio when the Tech/Civic popup appears" - it is clear that this option is only valid when there is a popup.

Reply:
Or shorter "Play audio with the Tech/Civic popup"

Reply:
Sounds good, will do